release.nix

```diff
@@ -45,7 +48,7 @@ let
   in pkgAttrs: builtins.concatLists (map extractDeps (builtins.attrValues pkgAttrs));
 
   perPlatform = lib.genAttrs cacheBuildSystems (system: let
-    obelisk = import ./. { inherit system; };
+    obelisk = import ./. (self-args // { inherit system; });
```

Comment:
I don't think so because we want to CI each platform. If you just want to CI one you should probably change `cacheBuildSystems` instead rather than forcing the attributes to lie.